ext/dvdread/dvdreadsrc.c

```diff
@@ -1051,6 +1060,14 @@ gst_dvd_read_src_set_property (GObject * object, guint prop_id,
         src->new_seek = TRUE;
       }
       break;
+    case ARG_STOP_AFTER_CHAPTER:
+      src->uri_stop_after_chapter = g_value_get_int (value);
+      src->stop_after_chapter = src->uri_stop_after_chapter - 1;
```

Comment:
I'm not exactly confident what the proper behavior should be.  I do not have the ability to test the situation where stop-after-chapter is modified during playback, but I predict that the change will not take effect until the read process reaches the end of the currently-playing chapter.